### PR TITLE
23 simplify compute metrics and plots

### PR DESCRIPTION
**Pull request by martinakaduc**
I cleaned the code.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -473,7 +452,7 @@ def run_exp(parms, env) -> None:
         "cost_fn": "euclidean",
         "plot": False,
         "gpu_id": 0,
-        "cont": False
+        "cont": False,
```
Comment (LMorlok) [inline main.py:455]: @martinakaduc is this running..? I think the comma is wrong there, right?

File: main.py [hunk 2]
```diff
@@ -431,35 +422,23 @@ def run_exp(parms, env) -> None:
...
-            "y": buffer["y"][i],
-            "cost": buffer["cost"][i],
-            "runtime": buffer["runtime"][i]
-        })
+        wandb.log(
+            {
+                "x": buffer["x"][i],
+                "y": buffer["y"][i],
+                "cost": buffer["cost"][i],
+                "runtime": buffer["runtime"][i],
```
Comment (LMorlok) [inline main.py:439]: same here. I think comma is wrong at the end there

File: main.py (hunk 1 above)
Comment (martinakaduc) [reply main.py:455]: These commas do not cause errors. If I remember correctly, when running formatting code with black, these commas are automatically added.

**Review by LMorlok — APPROVED** [review]
Comments are solved: Dictionary allows comma behind last element.

**Review by sangttruong — APPROVED** [review]
LGTM

**Pull request merged**